demos/pedestrian_tracker_demo/cpp/main.cpp

```diff
@@ -23,6 +22,13 @@
 #include <string>
 #include <gflags/gflags.h>
 
+#include <models/detection_model_centernet.h>
+#include <models/detection_model_ssd.h>
+#include <models/detection_model_yolo.h>
+#include "../../common/cpp/pipelines/include/pipelines/metadata.h"
+
+using namespace InferenceEngine;
```

Comment:
```suggestion
```
Please, add namespace directly where it is used, according to [the PR ](https://github.com/openvinotoolkit/open_model_zoo/pull/2661/files)

Reply:
done